### ISSUE TITLE
ELI5 explain prediction not working with DistillBert

### ISSUE THREAD

**Issue by KanishAnand** [amaiya/ktrain#234]
I am trying out the following code to train model 
``` python
MODEL_NAME = 'distilbert-base-uncased'
t = text.Transformer(MODEL_NAME, maxlen=MAX_LEN, classes=label_list)
trn = t.preprocess_train(train_text, train_tag)
val = t.preprocess_test(val_text, val_tag)
model = t.get_classifier()
learner = ktrain.get_learner(model, train_data=trn, val_data=val, batch_size=BATCH_SIZE)

history = learner.autofit(LR, EPOCHS)
predictor = ktrain.get_predictor(learner.model, preproc=t)
```
When I am trying to explain prediction using `predictor.explain` it's giving no results. And making direct call to eli5 library results in below error : 
![Screenshot from 2020-08-26 02-39-08](https://user-images.githubusercontent.com/43282223/91228183-70d52d00-e745-11ea-83a0-2e19184a21de.png)

Can you help me resolve issues of ktrain distill bert with eli5.

**Comment by KanishAnand** [comment]
I have following versions of libraries installed:
* ktrain: 0.20.0
* eli5: 0.10.1

**Comment by amaiya** [comment]
What version of `scikit-learn` are you using?  Are you sure you're using the latest `scikit-learn`?

As you can see from [this Colab notebook](https://colab.research.google.com/drive/1Grk6T-BN0JNyMGOVYMexdVwWwiIH8-w0?usp=sharing), everything works.

**Comment by KanishAnand** [comment]
My code was using same version of scikit-learn as that of mentioned Colab notebook. 
But I was running code on kaggle notebook and explain predictions was not working there. But when I tried same code on Google Colab that worked.
Thanks for help !

**Comment by amaiya** [comment]
Glad to hear you got it working.
